filebeat/input/filestream/config.go

```diff
@@ -33,6 +33,7 @@ import (
 type config struct {
 	Reader readerConfig `config:",inline"`
 
+	ID             string             `config:"id"`
```

Comment:
Need for better logging.

Reply:
This is an input ID in an input-level configuration. You can find it here https://www.elastic.co/guide/en/beats/filebeat/current/configuration-filebeat-options.html#CO11-1